include/wx/private/display.h

```diff
@@ -144,14 +154,22 @@ class wxDisplayImpl
     virtual bool ChangeMode(const wxVideoMode& mode) = 0;
 #endif // wxUSE_DISPLAY
 
+    // return true, if this display is still connected physically to system
+    virtual bool IsConnected() const { return m_isConnected; }
+
+    // indicate that this display is not connected to system anymore
+    virtual void Disconnect() { m_isConnected = false; }
+
 protected:
     // create the object providing access to the display with the given index
-    wxDisplayImpl(unsigned n) : m_index(n) { }
+    wxDisplayImpl(unsigned n) : m_index(n), m_isConnected(true) { }
 
 
     // the index of this display (0 is always the primary one)
-    const unsigned m_index;
+    unsigned m_index;
 
+    // true, if this display is still connected physically to system
+    bool m_isConnected;
```

Comment:
We prefer to initialize members in their declarations in the new code:
```suggestion
    bool m_isConnected = true;
```

(and then its initialization in the ctor above becomes unnecessary, of course)

Reply:
Fixed in 74f59d2.